Header header
bool success
string message
string command
uint64 dev_number
uint64 led_number
uint64 led_value
